src/main/java/org/broadinstitute/hellbender/tools/exome/HetPulldownCalculator.java

```diff
@@ -231,7 +236,7 @@ private Pulldown getHetPulldown(final File bamFile, final IntervalList snpInterv
             }
             logger.info("Examined " + totalNumberOfSNPs + " sites.");
             return hetPulldown;
-        } catch (final IOException e) {
+        } catch (final Exception e) {
```

Comment:
This is considered bad form to catch the Exception class...  Can you catch the specific types that are actually thrown?